subsys/bluetooth/mesh/dfd_srv.c

```diff
@@ -761,7 +761,7 @@ const struct bt_mesh_model_op _bt_mesh_dfd_srv_op[] = {
 	{ BT_MESH_DFD_OP_APPLY, BT_MESH_LEN_EXACT(0), handle_apply },
 	{ BT_MESH_DFD_OP_UPLOAD_GET, BT_MESH_LEN_EXACT(0), handle_upload_get },
 	{ BT_MESH_DFD_OP_UPLOAD_START, BT_MESH_LEN_MIN(16), handle_upload_start },
-	{ BT_MESH_DFD_OP_UPLOAD_START_OOB, BT_MESH_LEN_EXACT(2), handle_upload_start_oob },
+	{ BT_MESH_DFD_OP_UPLOAD_START_OOB, BT_MESH_LEN_MIN(2), handle_upload_start_oob },
```

Comment:
I guess commit message mistyping. It should be `This message _at least_ 2 bytes long, but can be longer, thus
BT_MESH_LEN_EXACT shouldn't be used.` or `This message _at least_ 2 bytes long, but can be longer, thus
BT_MESH_LEN_MIN should be used.`